scipy/interpolate/_rgi_cython.pyx

```diff
@@ -0,0 +1,126 @@
+# cython: language_level=3
+"""
+Cythonized routines for the RegularGridInterpolator.
+"""
+
+from libc.math cimport NAN
+import numpy as np
+cimport numpy as np
+cimport cython
+
+include "_poly_common.pxi"
+
+np.import_array()
+
+
+@cython.wraparound(False)
+@cython.boundscheck(False)
+@cython.initializedcheck(False)
+def evaluate_linear_2d(double_or_complex[:, :] values, # cannot declare as ::1
+                       long[:, :]indices,              # unless prior
```

Comment:
nitpick: there should be a space between `long[:, :]` and `indices`

Reply:
Thanks, done!